sdk/webpubsub/azure-messaging-webpubsubservice/examples/send_messages_connection_string_apim_proxy.py

```diff
@@ -0,0 +1,45 @@
+import io
+import logging
+import os
+
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.core.exceptions import HttpResponseError
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+try:
+    connection_string = os.environ['WEBPUBSUB_CONNECTION_STRING']
+    revers_proxy_endpoint = os.environ.get("WEBPUBSUB_REVERSE_RPOXY_ENDPOINT")
+except KeyError:
+    LOG.error("Missing environment variable 'WEBPUBSUB_CONNECTION_STRING' or 'WEBPUBSUB_REVERSE_RPOXY_ENDPOINT' - please set if before running the example")
+    exit()
+
+# Build a client from the connection string. And for this example, we have enabled debug
+# tracing. For production code, this should be turned off. 
+client = WebPubSubServiceClient.from_connection_string(connection_string, logging_enable=True, revers_proxy_endpoint=revers_proxy_endpoint)
+
+try:
+    # Raise an exception if the service rejected the call
+    client.web_pub_sub.send_to_all('Hub', message={'Hello': 'connection_string_reverse_proxy!'})
+    print('Successfully sent a JSON message')
+except HttpResponseError as e:
+    print('Failed to send JSON message: {}'.format(e.response.json()))
```

Comment:
also, I know the existing folder is called `examples`, but I'm thinking we should switch to `samples`, since that's what the other SDKs do. They also have a samples readme file, which lists all of the samples, and what they show. [Here's](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/textanalytics/azure-ai-textanalytics/samples) an example of an existing sdk